Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanGraphicsDevice.h

```diff
@@ -80,6 +84,11 @@ namespace webrtc
         VkCommandPool m_commandPool;
         VkCommandBuffer m_commandBuffer;
         VkFence m_fence;
+#else
+        static void AccessQueueCallback(int eventID, void* data);
```

Comment:
I think with Kazuki's revert, we also removed AccessQueueCallback.
This should be able to be removed right?

Reply:
Yup, it was removed in the original PR 966 but now that we try to keep both options (CRS vs separate cmdbuf/fence) open for additional testing and debugging, I added it back. I wasn't sure if there's any firm decisions either way for using AccessQueueCallback, but I can cherry-pick changesets here if needed or further tweaks could be another PR from Kazuki?

Reply:
I think we need to cherry pick / merge in the further changes from Kazuki during the revert.
The updated branch, "origin/fix/revert-vulkan", is how I think we can fix the original issue that lead to #966. 

I am pretty sure AccessQueue & the two part event was causing the issue that lead to the #966 fix, and the updated fix branch should work, but also needs testing to ensure that we don't hit issues trying to copy the resource Unity owns / controls with having exclusive queue access configured on the event.

Ideally, I think this branch should be a swap between the two if possible.

Reply:
This commit, "52cf51c7ad4bff35f0648fd0a4823772adcbc28b", is the one we have to port.
I think there is some additional clean-up we could do with AccessQueue removed, but can ignore that for now.

Reply:
Thanks! Just noticed the same changeset and branch, so crafted the QueueSubmit changes and AccessQueueCallback removal from there. I've never been a fan of kUnityVulkanEventConfigFlag_EnsurePreviousFrameSubmission with the CRS path so I left that one out, but once there's time to do more comparison and debugging, we can get back to it.